holidays/countries/bhutan.py

```diff
@@ -0,0 +1,104 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.groups import TibetanCalendarHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Bhutan(HolidayBase, TibetanCalendarHolidays):
+    """Bhutan holidays.
+
+    References:
+        * [ROYAL GOVERNMENT OF BHUTAN](https://web.archive.org/web/20250407140158/https://www.moha.gov.bt/wp-content/uploads/2023/11/National-Holiday-List-for-2024-2-.pdf)
+        * [PHPA-II HOLIDAY LIST 2025](https://phpa2.gov.bt/holiday-list/)
+    """
+
+    country = "BT"
+    start_year = 1970
+
+    def __init__(self, *args, **kwargs):
+        TibetanCalendarHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Winter Solstice.
+        self._add_holiday_jan_2("Winter Solstice")
+
+        # Traditional Day of Offering.
+        self._add_day_of_offering("Traditional Day of Offering")
+
+        # Losar.
+        self._add_losar("Losar")
+
+        name = "Birth Anniversary of His Majesty the King"
+        # Birth Anniversary of His Majesty the King (Day 1).
+        self._add_holiday_feb_21(name)
+
+        # Birth Anniversary of His Majesty the King (Day 2).
+        self._add_holiday_feb_22(name)
+
+        # Birth Anniversary of His Majesty the King (Day 3).
+        self._add_holiday_feb_23(name)
+
+        # Death Anniversary of Zhabdrung (Zhabdrung Kuchoe).
+        self._add_death_of_zhabdrung("Death Anniversary of Zhabdrung (Zhabdrung Kuchoe)")
+
+        # Birth Anniversary of the 3rd Druk Gyalpo.
+        self._add_holiday_may_2("Birth Anniversary of the 3rd Druk Gyalpo")
```

Comment:
Assuming this was added after Jigme Dorji Wangchuk's death
```suggestion
        if self._year >= 1973:
            # Birth Anniversary of the 3rd Druk Gyalpo.
            self._add_holiday_may_2("Birth Anniversary of the 3rd Druk Gyalpo")
```
If you're using 2007 as `start_year`, just ignore this comment.